applications/CompressiblePotentialFlowApplication/custom_conditions/adjoint_potential_wall_condition.cpp

```diff
@@ -68,23 +68,8 @@ void AdjointPotentialWallCondition<TPrimalCondition>::GetValuesVector(Vector& rV
     if(rValues.size() != TNumNodes)
         rValues.resize(TNumNodes, false);
 
-    bool is_kutta=false;
-    const auto& r_geometry = GetGeometry();
     for(unsigned int i=0; i<TNumNodes; i++){
-        if (r_geometry[i].GetValue(WAKE_DISTANCE)<0.0){
-            is_kutta=true;
-            break;
-        }
-    }
-    for(unsigned int i=0; i<TNumNodes; i++){
-        if(is_kutta){
-            if(r_geometry[i].GetValue(WAKE_DISTANCE)<0.0)
-                rValues[i] = GetGeometry()[i].FastGetSolutionStepValue(ADJOINT_VELOCITY_POTENTIAL);
-            else
-                rValues[i] = r_geometry[i].FastGetSolutionStepValue(ADJOINT_AUXILIARY_VELOCITY_POTENTIAL);
-        }
-        else
-            rValues[i] = r_geometry[i].FastGetSolutionStepValue(ADJOINT_VELOCITY_POTENTIAL);
+        rValues[i] = GetGeometry()[i].FastGetSolutionStepValue(ADJOINT_VELOCITY_POTENTIAL);
```

Comment:
This was doing nothing and Im removing it here.

Reply:
Ok